android/jni/com/mapswithme/maps/Framework.cpp

```diff
@@ -324,7 +324,7 @@ m2::PointD Framework::GetViewportCenter() const
   return m_work.GetViewportCenter();
 }
 
-void Framework::AddString(string const & name, string const & value)
+void Framework::AddString(std::string const & name, std::string const & value)
```

Comment:
using namespace std at the top of the file will significantly reduce the number of changes.